extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -324,8 +325,8 @@ export class RefreshManager {
   /**
    * Converts config to appropriate units, modifying the argument in place. This
    * also ensures that visiblePercentageMin is in the range of [0, 100].
-   * @param {!RefreshConfig} config
-   * @return {!RefreshConfig}
+   * @param {!JsonObject} config
+   * @return {!JsonObject}
```

Comment:
@erwinmombay I have one question, instead of returning a `JsonObject`, can we keep the `RefreshConfig` defination, but have the `convertAndSanitizeConfiguration_` returns an object like 
```
{
  continousTimeMin: config['continuousTimeMin']*1000,
  ...
}
```
I don't feel that the namespace `continousTimeMin` and `visiblePercentageMin` is useful after the read. WDYT?

Reply:
im ok with that for the most part.

just a thought though, using refreshconfig here, somebody is going to get tripped up again i think at some point. having a `typedef` does not mean closure will preserve the key values, and "configurations" almost always crosses these boundaries (xhr, html, postmessage, iframe) and the other side is going to get garbled obfuscated keys.

Reply:
> Thought typedef is more about type checking. If we move to JsonObject, is there a way to enforce type later?
This is why `JsonObject` should be used at the boundaries and types/typedefs everywhere else. unfortunately, its a little hard for me to interpret this much code so its easier for me to just `JsonObject` everything so that I don't make a mistake.

>I completely agree. I think this is a tradeoff between code size and style. Given this purely lives within extension code, I believe you're right to go with style.
did you mean you are OK with what I did, or do you prefer that i try to keep the `typedef`'s ?